modules/core/shared/src/main/scala/lucuma/core/enum/GpiObservingMode.scala

```diff
@@ -23,8 +22,8 @@ sealed abstract class GpiObservingMode(
   val apodizer:             Option[GpiApodizer],
   val fpm:                  Option[GpiFPM],
   val lyot:                 Option[GpiLyot],
-  val brightLimitPrism:     Option[BrightnessValue],
-  val brightLimitWollaston: Option[BrightnessValue],
+  val brightLimitPrism:     Option[BigDecimal],
+  val brightLimitWollaston: Option[BigDecimal],
```

Comment:
We could rename `BrightnessValue` to `MagnitudeValue` and keep it only for this case.

Reply:
I'm not sure I follow